app/i18n/locale_en.json

```diff
@@ -44,6 +44,8 @@
     "chart.title.pupil-teacher-ratio": "pupil teacher ratio",
     "chart.title.passrate-over-time": "pass rates over time",
     "chart.title.passrate-2014": "2014 pass rate",
+    "chart.title.rank-region": "REGIONAL RANK",
```

Comment:
I think we should keep our raw strings normal cased -- we can make any text ALL CAPS with CSS.